Reminder-App/Reminder-App/ReminderListDataSource.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ReminderListDataSource.swift
+//  Reminder-App
+//
+//  Created by BrainX Technologies 11 on 15/07/2021.
+//
+
+import UIKit
+
+class ReminderListDataSource: NSObject {
+    
+    private lazy var dateFormatter = RelativeDateTimeFormatter()
+}
+
+// MARK:- Data Source Methods
+extension ReminderListDataSource: UITableViewDataSource {
+    
+    // MARK:- Static Properties
+    static let reminderListCellIdentifier = "ReminderListCell"
+    
+    // Returns the number of rows to display (in table view section).
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return Reminder.testData.count
+    }
+    // Returns a cell for an index path.
```

Comment:
avoid unnecessary comments. your code should be self explanatory. so keep this in mind while naming your properties and functions